languages/en.php

```diff
@@ -619,35 +619,35 @@
 
 	'admin:options' => 'Admin options',
 
-	'admin:administer_security' => 'Security',
```

Comment:
security stuff was only part of master... i tried to keep the changes to a minimum